spec/unit/provider/package/dnf/python_helper_spec.rb

```diff
@@ -23,7 +23,7 @@
   let(:helper) { Chef::Provider::Package::Dnf::PythonHelper.instance }
 
   it "propagates stacktraces on stderr from the forked subprocess" do
-    allow(helper).to receive(:dnf_command).and_return("ruby -e 'raise \"your hands in the air\"'")
-    expect { helper.package_query(:whatprovides, "tcpdump") }.to raise_error(/your hands in the air/)
+    allow(helper).to receive(:query).and_raise("your hands in the air")
+    expect { helper.package_query("tcpdump", :whatprovides) }.to raise_error(/your hands in the air/)
```

Comment:
this is mocking out something that this was intended to test, that a raise from the actual shell command would get propagated.  the mock should still target dnf_command and not be changed.